oca_port/app.py

```diff
@@ -19,24 +18,18 @@ class App(Output):
 
     Parameters:
 
-        from_branch:
-            the source branch (e.g. '15.0')
-        to_branch:
-            the source branch (e.g. '16.0')
+        source:
+            SmartDict with branch, org, remote, source of the comparison
+        target:
+            SmartDict with branch, org, remote, target of the comparison
+        destination:
+            SmartDict with branch, org, remote, destination of the fork
```

Comment:
Hum I'm not a fan of depending on `make_gh_info/SmartDict` internals to initialize an `App` object when using oca-port as a Python package, I find it complicated from a user POV, especially for mandatory parameters.